content/when_you_die.md

```diff
@@ -4,78 +4,55 @@ description: What happens to your defined contribution pension when you die, inc
 
 # Your pension when you die
 
-The way you take your [defined contribution pension](/pension-types) may affect how much tax will be paid by the person you leave it to (your ‘beneficiary’) when you die.
+The way you take your pension will affect how you can leave it to your beneficiary (the person who inherits it) when you die.
 
-Most pension options allow anyone to inherit your pension. They don’t have to be a spouse or civil partner.
+Most pension options allow anyone to inherit your pension – they don’t have to be your spouse or civil partner.
 
-^Make sure your pension provider has up-to-date details for your beneficiary. If you have more than one pension, let all your providers know.^
+^Make sure your pension provider has up-to-date details of your beneficiary. If you have more than one pension, let all your providers know.^
 
 ## Leaving behind cash
 
-If you take your tax-free lump sum but don’t use it before you die (eg it’s left in your bank account), it becomes part of your estate. This then forms part of everything you own and all your money when you die. Your beneficiary may need to pay [Inheritance Tax](https://www.gov.uk/inheritance-tax/overview) on it.
+If you take your tax-free lump sum but don’t use it before you die (eg it’s left in your bank account), it becomes part of your estate. It then forms part of everything you own and all your money when you die. Your beneficiary may need to pay [Inheritance Tax](https://www.gov.uk/inheritance-tax) on it. 
 
-The same is true if you take your whole pot in one go or in chunks but don’t use it all before you die.
+The same is true if you take your whole pot in one go or in chunks but don’t use it all before you die. 
```

Comment:
I've just amended your original commit to take care of it. I'll merge when the build is green 👍